addons/spreadsheet/static/src/scss/spreadsheet_dashboard.scss

```diff
@@ -0,0 +1,3 @@
+.o-dashboard-opacity-25 .o-figure {
+    opacity: 0.25;
```

Comment:
I tried with 0.4 and it's better. There is less contrast between the titles and the graphs but still shows it's "fake" data.

Reply:
sure, I will make it 0.4